modules/serverless-build-func-kn.adoc

```diff
@@ -6,49 +6,42 @@
 [id="serverless-build-func-kn_{context}"]
 = Building functions
 
-Before you can run a function, you must build the function project by using the `kn func build` command. The build command reads the `func.yaml` file from the function project directory to determine the image name and registry.
+Before you can run a function, you must build the function project by using the `kn func build` command. The build command reads the `func.yaml` file from the function project directory to determine the image name and the registry used to store functions images.
```

Comment:
In the spirit of keeping func.yaml out of the examples as much as possible...
```suggestion
Before you can run a function, you must build the function project using the `kn func build` command. The build command uses the function project name and the image registry name to construct a fully qualified image name for your function.

.Example `func.yaml`
----
image: image-registry.openshift-image-registry.svc:5000/<oc_project>/<function_name>
----
```

Reply:
I'd remove "Example `func.yaml`" entirely then. I don't think just showing this one `image` line adds anything useful, really it just removes any context for users to see _how_ it fits into func.yaml.